src/app/js/admin/Appbar/AddFieldFromColumnButton.js

```diff
@@ -45,7 +45,7 @@ export const AddFieldFromColumnButtonComponent = ({
 AddFieldFromColumnButtonComponent.propTypes = {
     onShowExistingColumns: PropTypes.func.isRequired,
     p: polyglotPropTypes.isRequired,
-    scope: PropTypes.string.isRequired,
```

Comment:
Is this related to the change?

Reply:
No, but it was an error that went up in the console. So I took the opportunity to correct it